spring-boot-project/spring-boot-docs/src/main/asciidoc/howto.adoc

```diff
@@ -2564,7 +2564,7 @@ plugin, as shown in the following example:
 [source,groovy,indent=0]
 ----
 	plugins {
-		id "com.gorylenko.gradle-git-properties" version "1.4.17"
+		id "com.gorylenko.gradle-git-properties" version "1.5.0"
```

Comment:
The latest release seems to be `1.4.21`. The [Gradle portal](https://plugins.gradle.org/plugin/com.gorylenko.gradle-git-properties) provides the same info. Where is that `1.5.0` coming from?

Reply:
@snicoll  When I check in gradle portal it is 1.4.21 but from the original developer, it is 1.5.0 . hence changed to 1.5.0
Reference : https://github.com/n0mer/gradle-git-properties

Reply:
As far as I can tell, the README in that repository is incorrect. Version 1.5.0 has yet to be released. @rajadileepkolli Can you please update this PR accordingly? We'd also greatly appreciate it if you did some due diligence and checked that your proposed changes will work before submitting them.

Reply:
It's a version hardcoded in a README. it's not on the gradle portal, nor anywhere else as far as I can see. I've updated the doc to `1.4.21`.

Reply:
@wilkinsona , My bad it's my mistake, Initially I have tested locally with 1.4.21 and raised pull request with it only when providing the link I have misread and changed the value to 1.5.0.

I will take care that this won't happens again.